lib/htmlnano.es6

```diff
@@ -54,6 +61,18 @@ function htmlnano(optionsRun, presetRun) {
                 throw new Error('Module "' + moduleName + '" is not defined');
             }
 
+            (optionalDependencies[moduleName] || []).forEach(dependency => {
+                try {
+                    require(dependency);
+                } catch (e) {
+                    if (e.code === 'MODULE_NOT_FOUND') {
+                        console.warn(`You have to install "${dependency}" in order to use htmlnano's "${moduleName}" module`);
+                    } else {
+                        throw e;
+                    }
+                }
+            });
```

Comment:
Try to resolve the required dependencies before HTML actually being minified. With Node.js's require cache it won't add any performance overhead when those dependencies are required again.